### PR TITLE
Discover Work

### PR DESCRIPTION
**Pull request by enginnerd**
Updated styling to match home list items.
Only showing unique items.
Sorted descending order based on popularity (count).

Code clean-up still needed.

### CODE REVIEW

File: WealthNOW/.idea/gradle.xml [hunk 1]
```diff
@@ -6,6 +6,7 @@
         <option name="distributionType" value="LOCAL" />
```
Comment (mikeng13) [inline WealthNOW/.idea/gradle.xml]: Lol we should gitignore this.

File: WealthNOW/.idea/misc.xml [hunk 2]
```diff
@@ -3,8 +3,7 @@
   <component name="EntryPointsManager">
```
Comment (mikeng13) [inline WealthNOW/.idea/misc.xml]: And this

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/models/Stock.java [hunk 3]
```diff
@@ -51,6 +51,8 @@
     public String priceChange;
     @Column(name="ExDivDate")
     public Date exDivDate;
+    @Column(name="Popularity")
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/models/Stock.java]: I think we need a migration file this.

**Comment by enginnerd** [comment]
ok, i'll update .gitignore, merging branch

**Pull request merged**